### ISSUE TITLE
"Tips oss" på forsiden.

### ISSUE THREAD

**Issue by tonenordbo** [UX-Norge/ux-norge#64]
Lage en lignende denne litt høyt opp på forsiden:

![Image](https://user-images.githubusercontent.com/140154891/257161177-58aa5238-f0f5-4657-9d9b-1bbca32bf897.png)

**Comment by tonenordbo** [comment]
Vi kommer med design på denne.

**Comment by tonenordbo** [comment]
Design her:
https://www.figma.com/file/29x3Cw77bnX6YGYY1hcZMa/%F0%9F%8C%88-Nettside?type=design&node-id=337-989&mode=design&t=kkY107NRAZoUfZDV-4

**Comment by tonenordbo** [comment]
Skal være både på forsiden og inne på artikkelsidene.

**Comment by ksannaes** [comment]
Denne, ikke sant?

![Image](https://github.com/UX-Norge/ux-norge/assets/42838014/05f30939-b9bf-497a-a432-59e2d6dea0a3)

**Comment by mmoverla** [comment]
Jeg har lagt inn tips oss på forsiden og artikkelsiden,men før jeg comitter dette så lurer jeg på to ting:

- Skal knappen være en mailto-lenke?
- Dagens layout på artikkelsiden matcher ikke med layout på skissen, skal vi gjøre noe med det?

Skjermdump fra artikkelsiden lokalt:
![Image](https://github.com/UX-Norge/ux-norge/assets/5601800/69cc29ed-5701-41bc-ba9f-cd8febb79b9f)

**Comment by tonenordbo** [comment]
Ja, den kan være en mailto. Ja, vet at skissen ikke matcher, bare ignorer det :D Vi skal etterhvert redesigne toppen der, den har aldri funket helt.
